### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by hatboysam**

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting has begun.

To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must star this repo for your vote to count.**

A decision will be made after this PR has been open for **30** minutes, and at least **5** votes have been made.

*NOTE: the PR will be closed if any new commits are added after:* 291d668bd9b794db55151243d829a9009d5b40d0

**Comment by hatboysam** [comment]
:+1:

**Comment by Techwraith** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -13,18 +13,19 @@
...
   "keywords": [
     "nomic",
     "autonomous",
     "sentient",
-    "skynet"
+    "skynet",
+    "botwillacceptanything"
   ],
-  "author": "Matt Bell <mappum@gmail.com>",
+  "author": "Bot W. Acceptantyhing <botwillacceptanything@gmail.com>",
```
Comment (geoelectric) [inline package.json]: `Acceptantyhing` :/

**Comment by mappum** [comment]
Oops, I forgot to do that. Thanks! :+1:

**Comment by clay-to-n** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (hatboysam) [inline package.json]: Hmm I wish I could edit the PR without wiping out the current votes.  I will make that my next PR!

**Comment by ralston** [comment]
:+1:

**Comment by reportingsjr** [comment]
:+1:

**Comment by hamitron** [comment]
:+1:

**Comment by shalstvedt** [comment]
Ah, I'm late to this realization

**Comment by DavidTPate** [comment]
:+1:

**Comment by mostlygeek** [comment]
:+1:

**Comment by runvnc** [comment]
:+1:

**Comment by hatboysam** [comment]
It's been 35 minutes ... no merge.
@mappum

**Comment by mappum** [comment]
The bot is currently hitting the Github API rate limit :( Email Github and tell them to raise it for the bot's account!

**Comment by ravinggenius** [comment]
:+1:

**Comment by pjschlic** [comment]
:-1:

**Comment by generalhenry** [comment]
:+1:

**Comment by dos1** [comment]
:+1:

**Comment by botwillacceptanything** [comment]
#### :+1: The vote passed, this PR will now be merged into master.

**Pull request merged**